big_tests/tests/acc_e2e_SUITE.erl

```diff
@@ -127,6 +140,23 @@ message_altered_by_filter_local_packet_hook(Config) ->
             ok
         end).
 
+
```

Comment:
The test content doesn't imply there is some behind-the-scenes hook handling and dropping. I think we should have a short comment that describes what happens on MIM side.